support/client/lib/index.html

```diff
@@ -46,6 +46,7 @@
 
     <script type="text/javascript" src="alea.js"></script>
     <script type="text/javascript" src="mash.js"></script>
+    <script type="text/javascript" src="mespeak.js"></script>
```

Comment:
Can this move to the sound driver? The loader shouldn't be dependent on any particular driver.

The [cesium driver] (https://github.com/virtual-world-framework/vwf/blob/master/support/client/lib/vwf/model/cesium.js) has an example of loading the Cesium library this way:

    define( [ "module", "vwf/model", "vwf/utility", "vwf/utility/color", "vwf/model/cesium/Cesium" ],
        function( module, model, utility, Color, Cesium ) { ... } );